### DISCUSSION TITLE
Where can I get all combinations of hx-trigger ?

### DISCUSSION THREAD

**Discussion by kharabela** [bigskysoftware/htmx#2968] [Q&A]
Ex hx-trigger="every 1s"
I want to trigger something on focus out i.e  blur js equivalent.When user tabs out of the input field after some data is keyed

**Comment by Telroshan** [comment]
Not sure I fully understand the question, but you can refer to the [`hx-trigger`](https://htmx.org/attributes/hx-trigger/) documentation.
You can combine events and define several different triggers by using commas, for example : (example from the docs linked above)
```html
  <div hx-get="/news" hx-trigger="load, click delay:1s"></div>
```

In your case, you'll then want to simply use standard events such has `focusout`, maybe using the `from:` modifier if you want to trigger on a focusout on another element

Hope this helps!
